SlimApp/ClientCard.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ClientCard.swift
+//  SlimApp
+//
+//  Created by Aleksandr Shapovalov on 10/01/23.
+//
+
+import SwiftUI
+
+struct ClientCard: View {
+    @State private var showingSheet = false
+    var name: String = "Antonio Ferrari"
+    var nextTraining: String = "Today"
+
+    var body: some View {
+        Button {
+            showingSheet.toggle()
+        } label: {
+            HStack(spacing: 20) {
+                VStack(alignment: .leading, spacing: 8) {
+                    Text(name)
+                        .font(.system(size: 24, weight: .bold))
+                    Text(nextTraining)
+                        .font(.system(size: 18, weight: .regular))
+                }
+
+                Spacer()
+                Divider()
+                Image("4aces")
+                    .resizable()
+                    .scaledToFit()
+            }
+            .padding(20)
+            .frame(maxWidth: .infinity, maxHeight: 110)
```

Comment:
since you're using a Spacer(), wouldn't it be better to remove the infinity width? maybe just use a frame height

Reply:
agree, fixed it